controller/controller.go

```diff
@@ -270,20 +270,20 @@ func (c *Impl) processNextWorkItem() bool {
 	// resource to be synced.
 	if err = c.Reconciler.Reconcile(ctx, key); err != nil {
 		c.handleErr(err, key)
-		logger.Errorf("Reconcile failed. Time taken: %v.", time.Now().Sub(startTime))
+		logger.Infof("Reconcile failed. Time taken: %v.", time.Since(startTime))
```

Comment:
Change from Error to Info for reconcile failed?

Reply:
yep, this just logs the timing. `handleError` above, does the Error level error logging anyway.